conduit/pipeline/logging_test.go

```diff
@@ -29,6 +28,8 @@ func TestPluginLogFormatter_Format(t *testing.T) {
 	bytes, err := pluginFormatter.Format(entry)
 	assert.Nil(t, err)
 	str := string(bytes)
-	assert.Equal(t, str, "{\"__type\":\"A Question\",\"_name\":\"What's in a name?\",\"level\":\"info\",\"msg\":\"That which we call a rose by any other name would smell just as sweet.\",\"time\":\"0001-01-01T00:00:00Z\"}\n")
-
+	assert.Equal(t,
+		"{\"__type\":\"A Question\",\"_name\":\"What's in a name?\",\"level\":\"info\",\"msg\":\"That which we call a rose by any other name would smell just as sweet.\",\"time\":\"0001-01-01T00:00:00Z\"}\n",
```

Comment:
Interestingly, I'm _NOT_ actually changing this test in any meaningful way. The unit test already assumed this format and the zero-value of the timestamp doesn't print out the sub-second info.

Reply:
cf: https://go.dev/play/p/Jwj_N4ixBwy